apps/src/templates/studioHomepages/Courses.jsx

```diff
@@ -101,30 +78,18 @@ class Courses extends Component {
           <div className={'content'} style={styles.content}>
             <ProtectedStatefulDiv ref="flashes" />
 
-            {/* English, teacher.  (Also can be shown when signed out.) */}
-            {isEnglish && isTeacher && (
-              <div className={'announcements'}>
-                {specialAnnouncement && (
-                  <SpecialAnnouncementActionBlock
-                    announcement={specialAnnouncement}
-                  />
-                )}
-                <CoursesTeacherEnglish showAiCard={this.props.showAiCard} />
-              </div>
-            )}
-
-            {/* English, student.  (Also the default to be shown when signed out.) */}
-            {isEnglish && !isTeacher && (
+            {/* English */}
+            {isEnglish && (
               <div className={'announcements'}>
-                <SpecialAnnouncement isTeacher={isTeacher} />
+                <SpecialAnnouncement />
                 <CoursesStudentEnglish />
               </div>
             )}
 
             {/* Non-English */}
             {!isEnglish && (
               <CourseBlocksIntl
-                isTeacher={isTeacher}
+                isTeacher={false}
```

Comment:
`isTeacher` is required for `CourseBlocksIntl` so it is defaulting to the student view here

Reply:
Where else is CourseBlocksIntl used? Could we update it?

Reply:
Only used here! I'll update it